client/my-sites/pages/page.jsx

```diff
@@ -109,7 +112,12 @@ const Page = React.createClass( {
 
 	setAsHomepage: function() {
 		this.setState( { showPageActions: false } );
-		this.props.setFrontPage( this.props.page.site_ID, this.props.page.ID );
+		this.props.setFrontPage( this.props.page.site_ID, this.props.page.ID, function() {
+			// This gives us a means to fix the `SitesList` cache outside of actions
```

Comment:
I realize this callback is a workaround, but since it's duplicated in entirety, can we move it out to a utility function that gets imported so it's easier to pull out once the requisite `SitesList` work happens?